### PR TITLE
show: with `normalise` disable terminal width wrapping

### LINKED ISSUE #3751: opam show --normalise doesn't live up to its promise.
The `--normalise` option indicates it won't output newlines in values. Yet:

```shell
> opam --version
2.0.3
> opam show --normalise --no-lint --field=depends:,dev-repo: vg
depends:  ["ocaml" {>= "4.03.0"} "ocamlfind" {build} "ocamlbuild" {build} "topkg" {build} "gg" {>= "0.9.0"} "result" "uchar" "js_of_ocaml" {>= "3.0"} "js_of_ocaml-compiler" {>= "3.0"}
          "js_of_ocaml-ocamlbuild" {>= "3.0"} "js_of_ocaml-ppx" {>= "3.0"}]
dev-repo: "git+http://erratique.ch/repos/vg.git"
```
On the other hand if you ask for a single field it seems it doesn't: 
```shell
> opam show --no-lint --field=depends: --normalise vg 
["ocaml" {>= "4.03.0"} "ocamlfind" {build} "ocamlbuild" {build} "topkg" {build} "gg" {>= "0.9.0"} "result" "uchar" "js_of_ocaml" {>= "3.0"} "js_of_ocaml-compiler" {>= "3.0"} "js_of_ocaml-ocamlbuild" {>= "3.0"} "js_of_ocaml-ppx" {>= "3.0"}]
```

Sligthly related to https://github.com/ocaml/opam/issues/3415

### PR DESCRIPTION
**Pull request by rjbou**
closes #3751

### CODE REVIEW

File: src/client/opamListCommand.ml [hunk 1]
```diff
@@ -743,8 +743,13 @@ let info st ~fields ~raw_opam ~where ?normalise ?(show_empty=false) atoms =
           else acc)
         [] (List.rev fields)
     in
+    let cut =
```
Comment (AltGr) [inline src/client/opamListCommand.ml]: I believe you want `cut = None | Some `Truncate`, then `print_table ?cut`, to preserve the default wrapping when on a terminal

Comment (rjbou) [reply src/client/opamListCommand.ml]: updated

File: src/client/opamListCommand.ml [hunk 2]
```diff
@@ -743,8 +743,13 @@ let info st ~fields ~raw_opam ~where ?normalise ?(show_empty=false) atoms =
...
+      match normalise with
+      | Some n when n -> Some `None
```
Comment (AltGr) [inline src/client/opamListCommand.ml]: `| Some true ->` ;)

Comment (rjbou) [reply src/client/opamListCommand.ml]: updated :p

**Pull request merged**